### PR TITLE
[test] Add EMTEST_SKIP_NETWORK_TESTS

### PR DESCRIPTION
**Pull request by sbc100**
I was doing some work on an airplane over the weekend and I found this useful.

### CODE REVIEW

File: test/test_other.py [hunk 1]
```diff
@@ -2375,19 +2388,24 @@ def test_freetype(self):
     self.do_runf('freetype_test.c', expectedOutput,
                  emcc_args=['--use-port=freetype', '--embed-file', 'LiberationSansBold.ttf'])
 
+  @requires_network
+  @requires_network
```
Comment (kripken) [inline test/test_other.py:2392]:
```suggestion
```

**Review by kripken — COMMENTED** [review]
lgtm % fix

File: test/test_other.py (hunk 1 above)
Comment (kripken) [reply test/test_other.py:2392]: Looks like this fix was missed before landing?

Comment (kripken) [reply test/test_other.py:2392]: (there are two `@requires_network` for this function)

Comment (sbc100) [reply test/test_other.py:2392]: I pushed the submit button too quickly.   Fix was landed directly as a followup.

**Pull request merged**